src/deprecations.jl

```diff
@@ -5,3 +5,8 @@
 @deprecate takebit(index::T, bits::Int...) where T <: Integer  readbit(index, bits...)
 @deprecate breflect(nbits::Int, b::Integer)  breflect(b; nbits=nbits)
 @deprecate breflect(nbits::Int, b::T, masks::Vector{T}) where T breflect(b, masks; nbits=nbits)
+@deprecate bit(b; len) BitStr{len}(b)
+@deprecate to_location Base.to_index
```

Comment:
remove this, this is not correct. no need to deprecate `to_location` to `to_index`

Reply:
```suggestion
```